### ISSUE TITLE
Вагоны и поезда пропадают через месяц после появления

### ISSUE THREAD

**Issue by realbsb** [George-VB/xussrset#228]
Порой получается так, что даже и возить грузы не на чем
https://forums.ttdrussia.net/viewtopic.php?f=26&t=4701

**Comment by George-VB** [comment]
Загрузили сет паровозов? 
![image](https://github.com/George-VB/xussrset/assets/43310520/28ad6454-5028-44d7-9d32-ee3ec2fea6a9)
Использовали fast forward? Или какой-то модификатор скорости игры?

**Comment by realbsb** [comment]
Я рад, что это нестандартное поведение! Модификаторы скорости не использовал. Steamers Set загрузил:
![image](https://github.com/George-VB/xussrset/assets/3860342/b2ffd550-62f9-4004-b734-0067336f784c)

**Comment by realbsb** [comment]
Steamers Set загрузил.

**Comment by George-VB** [comment]
И всё равно нет паровозов?
Это полный перечень grf?

**Comment by realbsb** [comment]
> И всё равно нет паровозов? Это полный перечень grf?

Да, полный. Паровозы появляются буквально на месяц. Но есть некоторые, которые работают правильно. Точно не берусь перечислить, но среди них есть паровоз А, Щука (первая версия из них), Елена, Элька (первая версия из них).

**Comment by Nick222** [comment]
Я вообще не вижу большинство паровозов и почти всех вагонов (кроме вагона-ресторана 1932 года почему-то).
Активных файлов NewGRF - только тот самый адон с вагонами (версия 0.8.1r465).
Open TTD 14.1

**Comment by Nick222** [comment]
Примерно понятно, что происходит - отдельные ТС (вне групп и серий) показываются, а те, что в сериях или в группах - нет (видна одна группа и она не активна - серая).
